src/ui/graphics/lock_animation.h

```diff
@@ -0,0 +1,19 @@
+#ifndef _LOCK_ANIMATION_H
+#define _LOCK_ANIMATION_H
+
+/** ~3800ms unlock time, measured using this timer. */
+#define LOCK_ANIMATION_N_FRAMES (38)
+
+#define LOCK_ANIMATION_FRAME_WIDTH (28)
+#define LOCK_ANIMATION_FRAME_HEIGHT (25)
+#define LOCK_ANIMATION_FRAME_SIZE \
+    ((LOCK_ANIMATION_FRAME_WIDTH * LOCK_ANIMATION_FRAME_HEIGHT + 7) / 8)
+
+#include <stdint.h>
+
+/**
+ * Gets a frame of the lock animation.
+ */
+const uint8_t* get_lock_animation_frame(int frame_idx);
```

Comment:
nit: y u not like namespace prefixes :sweat_smile:   `lock_animation_get_frame`

Reply:
Ehm :| Fixed.

Reply:
(c7bdccc)